includes/WikiDiscover.php

```diff
@@ -448,6 +448,14 @@ public static function onParserGetVariableValueSwitch(
 
 				$ret = $cache[$magicWordId] = $dbr->selectRowCount( 'cw_wikis', 'wiki_url', [ 'wiki_deleted' => 0 ] );
 				break;
+			case 'numberofopenrequests':
+				$lbFactory = MediaWikiServices::getInstance()->getDBLoadBalancerFactory();
+
+				$dbr = $lbFactory->getMainLB( $config->get( '<>' ) )
+					->getMaintenanceConnectionRef( DB_REPLICA, [], $config->get( '<>' ) );
```

Comment:
uhh

Reply:
That was just a placeholder until I asked CA how to do that part.